Update release notes (#127)
frequenz-floss/frequenz-sdk-python: RELEASE_NOTES.md

## Summary

## Upgrading

- Add fine-grained Result types for the PowerDistributingActor.
    Previously Result was one class with many fields. Now each result has its own class
    that derives from Result parent class.

## New Features

> - Add inverter type to the Component data.
>     Inverter component has type that tells what kind of
>     inverter it is (Battery, Solar etc).

Reviewer: In the release notes is better not to do manual wrapping because in the GitHub release section, damn GitHub will add a line break for each `\n`, so then it looks weird. Not sure if the extra newline I'm suggesting is necessary (or the 2 spaces indent), if you checked it and is fine like it is you can ignore it.

```suggestion
- Add inverter type to the Component data.

  Inverter component has type that tells what kind of inverter it is (Battery, Solar etc).
```

Author: I just change it to one line description (as it was made in last release). :)

- Add `FormulaGenerator` class for generating formulas from the component graph.

> - Add formulas for: grid_power, battery_power, PV power.

Reviewer: ```suggestion
- Add formulas for: `grid_power`, `battery_power`, `pv_power`.
```

Author: Done

## Bug Fixes

> - Fix ComponentMetricResamplingActor, to don't subcribe twice to the same source.

Reviewer 2: ```suggestion
- Fix ComponentMetricResamplingActor, to not subscribe twice to the same source.
```

Author: Done
